pkg/kubernetes/kubernetes.go

```diff
@@ -20,16 +20,25 @@ import (
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Init deploys the Dapr operator
-func Init() error {
+const (
+	daprLatestVersion = "latest"
+)
+
+// Init deploys the Dapr operator using the supplied runtime version.
+func Init(version string) error {
 	client, err := DaprClient()
 	if err != nil {
 		return fmt.Errorf("can't connect to a Kubernetes cluster: %v", err)
 	}
 
-	version, err := cli_ver.GetLatestRelease(cli_ver.DaprGitHubOrg, cli_ver.DaprGitHubRepo)
-	if err != nil {
-		return fmt.Errorf("cannot get the manifest file: %s", err)
+	if version == daprLatestVersion {
+		var v string
+		v, err = cli_ver.GetLatestRelease(cli_ver.DaprGitHubOrg, cli_ver.DaprGitHubRepo)
```

Comment:
can it be refactored, this call should happen only when version is not latest, in all other cases it should just use the user provided version

Reply:
> can it be refactored, this call should happen only when version is not latest, in all other cases it should just use the user provided version

The default version is always "latest" if not provided by user, so this block will get the latest release version number and use it to fetch respective latest versioned dapr-operator.yaml..

Reply:
in this case, will this call to "cli_ver.GetLatestRelease" be made when user provides a version?

Reply:
> in this case, will this call to "cli_ver.GetLatestRelease" be made when user provides a version?

no. user provided version will directly be used in fetching dapr_operator.yaml at Line 44.